includes/class-wc-tracker.php

```diff
@@ -366,16 +366,96 @@ private static function get_order_counts() {
 	}
 
 	/**
-	 * Combine all order data.
+	 * Get all order data.
 	 *
 	 * @return array
 	 */
 	private static function get_orders() {
-		$order_dates  = self::get_order_dates();
-		$order_counts = self::get_order_counts();
-		$order_totals = self::get_order_totals();
+		$orders = wc_get_orders(
+			array(
+				'type'           => array( 'shop_order', 'shop_order_refund' ),
+				'customer'       => '',
+				'posts_per_page' => -1,
```

Comment:
I'm slightly worried here in terms of memory usage on large shops. Have you tried to run this on a store with tens of thousands of orders? It might be better to paginate & iterate over it, what do you think?

Reply:
Ah yes, I haven't tested it in large stores. What is the standard page size that is used in WC? I'll use the same.